### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: vscode.opam [hunk 1]
```diff
@@ -10,10 +10,11 @@ doc: "https://tmattio.github.io/js-bindings/"
 bug-reports: "https://github.com/tmattio/js-bindings/issues"
 depends: [
   "ocaml" {>= "4.08.0"}
-  "dune"
-  "ojs"
   "es5" {= version}
+  "dune"
+  "gen_js_api"
```
Comment (tmattio) [inline vscode.opam]: I purposefully avoided the dependency to `gen_js_api` in the opam packages. It's used only during development to generate the implementations, and the files are then promoted. We can install it manually in the CI

Comment (smorimoto) [reply vscode.opam]: Indeed, I just changed that.

**Comment by smorimoto** [comment]
By the way, there is no package that matches the name field of the dune-project, so trying to pin and install will fail.

**Comment by smorimoto** [comment]
The macOS build is a bit slow, but this is probably due to the problem I mentioned in the v2 PR. v2 is still based on the unmerged PR, but is well tested and generally ready for use. It includes features for large repositories and fixes for Windows.

**Comment by smorimoto** [comment]
See here for more info: https://github.com/avsm/setup-ocaml/pull/66

**Comment by smorimoto** [comment]
I just enabled dune cache on Windows as well.

**Comment by tmattio** [comment]
Thanks a lot 🎉

**Pull request merged**